### PR TITLE
[Test] USWDS-Sandbox: Use uswds 2.14.0 and compile 1.2.0

### PR DESCRIPTION
**Pull request by amyleadem**
## Summary
Created an implementation that both uses USWDS 2 and @uswds/compile  1.2.0 so that we can test the compile release with both major versions of the design system.

> [!important]
> This PR is for testing purposes only and should not be merged into main.

> [!Tip]
> We could use this PR to test future releases of uswds compile with uswds v2. We just need to update the version of `@uswds/compile in package.json` (or just do a clean install if the release is in `develop`.

## Major changes
1. Replaced `@uswds/uswds 3.8.1` dependency with `uswds 2.14.0`
1. Copied over the `/_sass` directory from `develop` into a new `src/sass` directory
1. Updated gulpfile  to use `uswds.settings.version = 2;` and `uswds.paths.dist.theme = "./src/sass";`
1. Replaced `uswds/compile 1.1.0` dependency with uswds compile `develop` branch for testing

## Testing and review
1. Open this branch locally
2. Run a clean npm install
3. Run `npx gulp init`
4. Run gulp tasks from `gulpfile.js` like `npx gulp watchSass`, `npx gulp compileIcons` and confirm there are no errors

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -12,10 +12,15 @@ PATHS
...
+// Comment out "Version 3" section below when this section is active
+uswds.settings.version = 2;
+uswds.paths.dist.theme = "./src/sass";
 
-// Project Sass source directory
-uswds.paths.dist.theme = "./src/_styles";
+// Version 3 settings:
+// Comment "Version 2" section above when this section is active
+// uswds.settings.version = 3;
+// uswds.paths.dist.theme = "./src/_styles";
```
Comment (amyleadem) [inline gulpfile.js:23]: > [!note]
> For testing purposes, I included Sass for both version 3 and version 2 in the project. We'll need to comment out the version 3 section when the version 2 section is active and vice versa.

**Comment by mejiaj** [comment]
Confirming, no issues found on `npm install`, `npx gulp init`, and `npx gulp buildSass`.

**Review by mejiaj — APPROVED** [review]
Confirming, no issues found on `npm install`, `npx gulp init`, and `npx gulp buildSass`.

**Review by mahoneycm — APPROVED** [review]
Works like a charm! I was able to run through all gulp commands without an issue.

**praise:** Thanks for setting this up! It'll really come in handy for testing purposes

**Comment by mejiaj** [comment]
**Thought**: We should create a branch for USWDS 2 testing and merge this into that for future testing.

**Review by mejiaj — APPROVED** [review]
Thanks for creating this. 

I've created a `v2-main` branch we can merge this into and re-use for future testing. This way we can close this PR and have a branch ready to develop from.

**Pull request merged**